viewer/src/data/network/Semaphore.ts

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright 2020 Cognite AS
+ */
+
+export class Semaphore {
```

Comment:
Should this be moved to `src/utils/`?

Reply:
Not the biggest fan of utils-folders if we can find a name that better describes something. Any thoughts @j-bjorne ?

Reply:
I would want it to be called something like JobScheduler, or JobHandler / Limiter and moved into data/workers or data/utils?